.buildkite/build_pex.sh

```diff
@@ -2,7 +2,7 @@
 
 set -euo pipefail
 
-pip install pex                 # pex is really the only thing we need here.
+pip install pex\<1.3  # pex is really the only thing we need here.
```

Comment:
is the `\` slash here a typo?

Reply:
No it's intentional, the `<` has special meaning in a shell

Reply:
(it pipes the contents of a file into the stdin of a command)

Reply:
got it, thanks!

might recommend conventionally using quotes for readability? e.g. `pip install "pex<1.3"`